highfidelity/hifi
# Fix AC recording playback audio encoding

Fixes the audio encoding of a scripted agent that plays back a recording.

## QA Test
- Run the Sandbox of this PR
- Make the recording using Interface
   1. Use the `developer/utilities/record/recorder.js` script to make a simple recording while the microphone is  not muted. 
   2. Save the file to disk
- Run the scripted agent:
    1. Go to your local `Create JS Assignment` page (http://localhost:40100/assignment/# )
    2. Copy paste the contents of the following script in the page: 
https://raw.githubusercontent.com/highfidelity/hifi/d83600a22ee1579b15acf742e2fa83a84ce9951c/script-archive/acScripts/PlayRecordingOnAC.js
    3. Replace `http://your.recording.url` with the `file:///` path  of your recording (e.g. `file:///C:/Users/Thijs/Desktop/testThisOnAC.hfr`)
    4. Leave the `Pool` field empty and set `# of Instances` to `1` and press the Run button
- Go back into Interface and verify that you can hear the audio of the recording playing.
